projects/ngx-sub-form/src/lib/ngx-sub-form-utils.ts

```diff
@@ -20,11 +20,14 @@ export type ControlsNames<T> = { [K in keyof T]-?: K };
 export type ControlMap<T, V> = { [K in keyof T]-?: V };
 
 export type ControlsType<T> = { [K in keyof T]-?: T[K] extends any[] ? FormArray : AbstractControl };
+export type FormErrorsType<T> = { [K in keyof T]-?: T[K] extends any[] ? ValidationErrors[] : ValidationErrors };
 
 export type FormUpdate<FormInterface> = { [FormControlInterface in keyof FormInterface]?: true };
 
 export type FormErrors<FormInterface> = null | Partial<
-  ControlMap<FormInterface, ValidationErrors | ValidationErrors[] | null> & { formGroup?: ValidationErrors }
+  FormErrorsType<FormInterface> & {
+    formGroup?: ValidationErrors | null | undefined;
```

Comment:
the `null | undefined` is not correct - if you check the implementation it is impossible that the key could be present and null or undefined

Reply:
Indeed. Fixed